_posts/2019-11-19-apple-tv-plus.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Apple TV Plus
+date: 2019-11-19 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I've finally tried Apple TV Plus for a few weeks and I can say that the shows are good, but the software is terrible.
+
+I'm only interested in Saw and For All Mankind so I did not watch anything else, but both are great and well worth a watch if you like the premises of the series.
```

Comment:
Saw - > See

Reply:
Premise - > genre?

Reply:
more than gender is the topic their talk about, so I think premise is more accurate